### PR TITLE
New package: sigoden.AIChat version 0.25.0

### PR DESCRIPTION
**Pull request by SpecterShell**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/203033)

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

File: manifests/s/sigoden/AIChat/0.25.0/sigoden.AIChat.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- aichat
+ReleaseDate: 2024-12-10
+Installers:
+- Architecture: x86
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: aichat.exe
+    PortableCommandAlias: aichat
+  InstallerUrl: https://github.com/sigoden/aichat/releases/download/v0.25.0/aichat-v0.25.0-i686-pc-windows-msvc.zip
+  InstallerSha256: 5267ACAC78921DC924C41CB59964F225A472242EC2DC0F7FBCBCC59DF1456ED9
```
Comment (stephengillie) [inline manifests/s/sigoden/AIChat/0.25.0/sigoden.AIChat.installer.yaml:17]:
```suggestion
  InstallerSha256: 5267ACAC78921DC924C41CB59964F225A472242EC2DC0F7FBCBCC59DF1456ED9
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```

File: manifests/s/sigoden/AIChat/0.25.0/sigoden.AIChat.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+- Architecture: x64
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: aichat.exe
+    PortableCommandAlias: aichat
+  InstallerUrl: https://github.com/sigoden/aichat/releases/download/v0.25.0/aichat-v0.25.0-x86_64-pc-windows-msvc.zip
+  InstallerSha256: A0667829A43E623E03B445DD436C0D765D5BBE6E4F9D7E4E728950007556BF58
```
Comment (stephengillie) [inline manifests/s/sigoden/AIChat/0.25.0/sigoden.AIChat.installer.yaml:24]:
```suggestion
  InstallerSha256: A0667829A43E623E03B445DD436C0D765D5BBE6E4F9D7E4E728950007556BF58
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

File: manifests/s/sigoden/AIChat/0.25.0/sigoden.AIChat.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+- Architecture: arm64
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: aichat.exe
+    PortableCommandAlias: aichat
+  InstallerUrl: https://github.com/sigoden/aichat/releases/download/v0.25.0/aichat-v0.25.0-aarch64-pc-windows-msvc.zip
+  InstallerSha256: 6A141A30AFAF0842CBBF04F47BC0854B06E5FE1A74A11A2B1EEA282419028E4C
```
Comment (stephengillie) [inline manifests/s/sigoden/AIChat/0.25.0/sigoden.AIChat.installer.yaml:31]:
```suggestion
  InstallerSha256: 6A141A30AFAF0842CBBF04F47BC0854B06E5FE1A74A11A2B1EEA282419028E4C
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.arm64
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
image

**Review by ItzLevvie — APPROVED** [review]

**Comment by stephengillie** [comment]
Verify:
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**